jina/orchestrate/deployments/config/docker_compose.py

```diff
@@ -51,11 +51,12 @@ def get_gateway_config(
             import os
 
             test_pip = os.getenv('JINA_K8S_USE_TEST_PIP') is not None
```

Comment:
This shouldn't be used anymore, right? Rather set `JINA_GATEWAY_IMAGE` in CI?

Reply:
good point, will change